plugins/aladino/actions/review.go

```diff
@@ -21,17 +23,39 @@ func Review() *aladino.BuiltInAction {
 
 func reviewCode(e aladino.Env, args []aladino.Value) error {
 	t := e.GetTarget().(*target.PullRequestTarget)
+	reviewpadBot := "reviewpad-bot"
 
 	reviewEvent, err := parseReviewEvent(args[0].(*aladino.StringValue).Val)
 	if err != nil {
 		return err
 	}
 
-	reviewBody, err := checkReviewBody(reviewEvent, args[1].(*aladino.StringValue).Val)
+	reviewBody := args[1].(*aladino.StringValue).Val
+
+	// Only a review with APPROVE does not need a review comment
+	if reviewEvent != "APPROVE" && reviewBody == "" {
+		return fmt.Errorf("review: comment required in %v event", reviewEvent)
+	}
+
+	reviews, err := t.GetReviews()
 	if err != nil {
 		return err
 	}
 
+	// If the last review of reviewpad-bot was an approval or if there were no updates to the pull request
+	// since last review made by reviewpad-bot, then do nothing.
+	if codehost.HasReview(reviews, reviewpadBot) {
+		lastReview := codehost.LastReview(reviews, reviewpadBot)
+
+		if lastReview.State == "APPROVED" {
+			return nil
+		}
+
+		if !lastReview.SubmittedAt.Before(t.PullRequest.GetUpdatedAt()) {
```

Comment:
```suggestion
		if lastReview.SubmittedAt.After(t.PullRequest.GetUpdatedAt()) {
```

Replace `! Before` with `After` for better reading.